UPGRADING.md

```diff
@@ -176,49 +185,44 @@ The following options have been added to the `BugsnagEvent` class:
+| `user` / `setUser:withEmail:andName` / `setUser(_:email:name)`     | The user of the app when the event occurred. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#setuser) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#setuser) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#setuser).)
 
 #### Changes
 
-Event data is now made available as structured classes on the `device`, `app`, `errors` and `threads` fields on the `BugsnagEvent`, previously this was included in undocumented `NSDictionary` data. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#the-bugsnagevent-class) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#the-bugsnagevent-class) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#the-bugsnagevent-class)).
+Event data is now made available as structured classes on the `device`, `app`, `errors` and `threads` fields on the `BugsnagEvent`, previously this was included in undocumented `NSDictionary` data. (See docs: [iOS](https://docs.bugsnag.com/platforms/ios/customizing-error-reports/#the-bugsnagevent-class) \| [macOS](https://docs.bugsnag.com/platforms/macos/customizing-error-reports/#the-bugsnagevent-class) \| [tvOS](https://docs.bugsnag.com/platforms/tvos/customizing-error-reports/#the-bugsnagevent-class).)
 
 Each event is now also delivered in a separate request to avoid exceeding Bugsnag's request payload size limit in extreme scenarios.
 
 In addition:
 
-##### Objective-C
-
-| v5.x API                                                           | v6.x API                                                          |
-| ------------------------------------------------------------------ | ----------------------------------------------------------------- |
-| `addAttribute:withValue:toTabWithName:`                            | `addMetadataToSection:withKey:toSection:`                         |
-
-##### Swift
-
 | v5.x API                                                           | v6.x API                                                          |
 | ------------------------------------------------------------------ | ----------------------------------------------------------------- |
-| `addAttribute(attributeName:withValue:toTabName`                   | `addMetadata(metadata:key:section)`                               |
+| `addAttribute:withValue:toTabWithName:` /<br/> `addAttribute(attributeName:withValue:toTabName)` | `addMetadata:withKey:toSection:` / <br />`addMetadata(metadata:key:section)` |
```

Comment:
We're missing the `addMetadata:toSection` method here.

Reply:
That's an addition - it's in the table above. You could previously add individual elements but not the whole map.